examples/getting-started-movielens/03-Training-with-PyTorch.ipynb

```diff
@@ -37,52 +37,44 @@
     "\n",
```

Comment:
<p>I am not sure this <code>The second tensor is a supporting Tensor, describing how many genres are associated with each datapoint in the batch. </code> sentence is really accurate. why because  according to my understanding from this sentence,  I see that second tensor has <code>88846</code> value and but a batch  in this example cannot have a single tensor of that amount of data points.   </p><p> </p><p>I'd say the second tensor is representing the starting point of each tensor of genres, and the difference between two successive number shows the number of datapoints in each tensor. </p><p> </p><p>basically, can we clarify it better? what do you think? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Absolutely! Made the change as requested</p>